runtime/drive_handler_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStubDriveHandler(t *testing.T) {
+	const tempPath = "test"
+	err := os.Mkdir(tempPath, os.ModePerm)
+	assert.NoError(t, err)
+	defer func() {
+		os.RemoveAll(tempPath)
+	}()
+
+	handler := newStubDriveHandler(tempPath)
+	paths, err := handler.StubDrivePaths(5)
+	assert.NoError(t, err)
+	assert.Equal(t, 5, len(paths))
+
+	infos, err := ioutil.ReadDir(tempPath)
+	assert.NoError(t, err)
+	assert.Equal(t, 5, len(infos))
+}
```

Comment:
Can you add some tests for `PatchStubDrives`?  It'd be good to have a simple one that verifies basic functionality as well as one that tests the concurrency logic in there (your use of `atomic.LoadInt64` and `atomic.AddInt64`).